dolphin/db/sqlalchemy/api.py

```diff
@@ -93,6 +95,8 @@ def access_info_get(context, storage_id):
     access_info = this_session.query(AccessInfo) \
         .filter(AccessInfo.storage_id == storage_id) \
         .first()
+    if not access_info:
+        raise exception.ResourceNotFound(storage_id=storage_id)
```

Comment:
suggest here:
```
msg = _("Storage access info %s could not be found.") % storage_id
raise exception.NotFound(msg)
```
or change ResourceNotFound to AccessInfoNotFound and update the message of it.
```
message = _("Storage access info %(storage_id)s could not be found.")